pkg/operator/ceph/csi/template/csi-logrotate-sidecar.yaml

```diff
@@ -29,7 +39,7 @@ image: {{ .CSIPluginImage }}
 imagePullPolicy: IfNotPresent
 name: log-collector
 volumeMounts:
-  - mountPath: {{ .CsiLogDirPath }}/cephcsi/logrotate-config/{{ .CSILogFolder }}
+  - mountPath: {{ .CsiLogDirPath }}/{{ .DriverNamePrefix }}/cephcsi/logrotate-config/{{ .CSILogFolder }}
```

Comment:
This doesn't seem consistent with the rest of the changes, it's missing the full provider name

Reply:
THanks for catching, done